src/main/webapp/i18n/de/exercise.json

```diff
@@ -181,13 +181,15 @@
             "isTeamExercise": "Diese Aufgabe ist eine Teamaufgabe.",
             "update": {
                 "warning": {
-                    "warning": "Warnung!",
-                    "confirm": "Änderungen bestätigen",
-                    "problems": "Folgende Probleme können auftreten:",
-                    "instructionDeleted": "Benotungshinweise gelöscht:",
-                    "scoringChange": "Bewertungsart verändert:",
-                    "results": "Die bestehenden Ergebnisse müssen neu berechnet werden!",
-                    "confirmtext": "Möchtest du die Änderungen wirklich übernehmen?"
+                    "warning": "Warnung: Diese Änderung wird möglicherweise Probleme verursachen",
+                    "saveExerciseWithoutReevaluation": "Nicht neu bewerten",
+                    "problems": "Um Inkonsistenzen zwischen bestehenden Rückmeldungen und zukünftigen Rückmeldungen zu vermeiden, die mit der betroffenen Benotungsanweisung verbunden sind, muss die Übung neu bewertet werden. Dazu gehört auch die Neuberechnung der Ergebnisse.",
```

Comment:
Almost 😄 We use "feedback" instead of "Rückmeldung" (eventhough Rückmeldung would be the translation :)) and "Benotungshinweis" (not Anweisung) for grading instruction.
-> "problems": "Um Inkonsistenzen zwischen bestehendem und zukünftigem Feedback, das mit der betroffenen Benotungshinweis verbunden ist, zu vermeiden, muss die Aufgabe neu bewertet werden. Dazu gehört auch die Neuberechnung der Ergebnisse."